source/renderer/app/components/layout/TopBarLayout.scss

```diff
@@ -19,5 +19,5 @@ $topBarHeight: 84px;
 }
 
 .content {
-  height: calc(100% - #{$topBarHeight});
+  height: auto;
```

Comment:
@yakovkaravelov are you sure this is right? Please check how the screens which have little content look like. I am afraid the `.content` wrapper will not extend to the available height in this case.